bin/notification.py

```diff
@@ -64,6 +87,60 @@ def send_slack_to_user(self, user_id, text):
             logging.error(
                 f'Unexpected response code from slack api when opening conversation with user {user_id}: {response}')
 
+
+class AnomalyDetectionNotificator(Notificator):
+    """
+    Handles messages form CloudWatch about abnormally large (for the given user) athena queries
+    """
+
+    def __init__(self, config):
+        self.config = config
+
+    def handle_single_event(self, record):
+        event = AnomalyDetectionEvent(**record)
+        record_sns = AnomalyDetectionSns(**event.Sns)
+        subject = record_sns.Subject
+        message = AnomalyDetectionMessage(record_sns.Message)
+        new_state_value = message.new_state_value
+        if str.lower(new_state_value) == 'alarm':
+            new_state_reason = message.new_state_reason
+            user = message.user
+            params = dict(subject=subject, user=user, new_state_reason=new_state_reason)
+            text = self.config.SLACK_ANOMALY_DETECTION_THRESHOLD_MESSAGE.format(**params)
+            if hasattr(self.config, 'SLACK_USER_MAPPINGS'):
+                slack_user = self.config.SLACK_USER_MAPPINGS.get(user)
+            self.send_slack_to_channel(text)
```

Comment:
not very important, but can you move this line above the hasattr check? When you do it, we will have the code that sends messages to slack channel separated from the code that deals with sending messages to individual users